pkg/services/ngalert/schedule/registry_test.go

```diff
@@ -239,7 +239,7 @@ func TestSchedulableAlertRulesRegistry(t *testing.T) {
 	assert.Len(t, rules, 0)
 	assert.Len(t, folders, 0)
 
-	expectedFolders := map[models.FolderKey]string{models.FolderKey{OrgID: 1, UID: "test-uid"}: "test-title"}
+	expectedFolders := map[models.FolderKey]string{{OrgID: 1, UID: "test-uid"}: "test-title"}
```

Comment:
this seems unrelated...

Reply:
👍 it looks like this came from [021d2cc](https://github.com/grafana/grafana/pull/81591/commits/021d2cc7e52403c5693bbf21d3fa3e2b0e90d374). i'll revert